migrationmanager/migrators/logs/migrations/000005_attribute_rename.down.sql

```diff
@@ -1,32 +1,32 @@
-RENAME TABLE IF EXISTS signoz_logs.logs_attribute_keys TO signoz_logs.logs_atrribute_keys on CLUSTER cluster;
+RENAME TABLE IF EXISTS signoz_logs.logs_attribute_keys TO signoz_logs.logs_atrribute_keys ON CLUSTER {{.SIGNOZ_CLUSTER}};
```

Comment:
Please add a line from here https://github.com/SigNoz/signoz-otel-collector/blob/5758bc46545f82997beaf43faa3c4cfa7f428584/exporter/clickhouselogsexporter/migrations/000005_attribute_rename.up.sql#L2 as it was added recently.